scripts/yalc-sm.ts

```diff
@@ -0,0 +1,64 @@
+import { existsSync, readdirSync } from "node:fs";
```

Comment:
⚠️ For me it would be perfect to a small and simple bloc in the CONTRIBUTING.md guide about that. 
Like "How to work locally with editor packages?" something like that.

Reply:
Done!